testproject/Assets/Tests/Runtime/MultiprocessRuntime/Helpers/BuildMultiprocessTestPlayer.cs

```diff
@@ -109,6 +109,7 @@ public struct BuildInfo
 
         public static BuildInfo ReadBuildInfo()
         {
+            //TODO: For mobile platforms, determine if this is the right way to get the BuildInfoFile, currently this fails on android
```

Comment:
your PR says "enable testing in CI for mobile" does that mean this PR only works on iOS?

Reply:
Multiprocess tests are not currently in scope for this PR, I'll have to come back around to it.

Reply:
At the moment, I more focused on Android, I'll do iOS in a separate PR but much of the work is effectively the same or at least, very similar.